OpenSim/Simulation/Model/PhysicalFrame.cpp

```diff
@@ -48,7 +48,7 @@ PhysicalFrame::PhysicalFrame() : Frame()
     FrameGeometry frm(0.2);
```

Comment:
Agreed if we can provide customization since otherwise all frames show the same and if on top of each other are practically useless. Outside the scope of this PR I believe.